cookbooks/cdo-varnish/libraries/helpers.rb

```diff
@@ -146,13 +146,13 @@ def process_request(behavior, _)
   out = ''
   cookies = behavior[:cookies]
   out << case cookies
-    when 'all'
-      '# Allow all request cookies.'
-    when 'none'
-      'cookie.filter_except("NO_CACHE");'
-    else
-      cookies.map{ |c| extract_cookie(c)}.join + "cookie.filter_except(\"#{cookies.join(',')}\");"
-  end
+         when 'all'
+           '# Allow all request cookies.'
+         when 'none'
+           'cookie.filter_except("NO_CACHE");'
+         else
+           cookies.map{ |c| extract_cookie(c)}.join + "cookie.filter_except(\"#{cookies.join(',')}\");"
+         end
```

Comment:
I like the following a little better (doesn't indent as far if the start of the line gets super long), but what you have is also fine:
```ruby
out <<
  case cookies
  when 'all'
   '# Allow all request cookies.'
  when 'none'
   'cookie.filter_except("NO_CACHE");'
  else
   cookies.map{ |c| extract_cookie(c)}.join + "cookie.filter_except(\"#{cookies.join(',')}\");"
  end
```